src/game/generation/RegionGenerator.java

```diff
@@ -93,6 +93,9 @@ public void generateWorld(int x, int y) {
 		}
 	}
 
+	private static final int SEALEVEL = (int) (CHUNK_HEIGHT * 0.25)
+			+ getAmplitude(BiomeType.OCEAN) + 1;
```

Comment:
It's generally bad practice to define instance variables or constants in the middle of the file. I'd move this to the top, so that it would be easier for other people (and yourself) to find the value.

Reply:
Will you or should I?

Reply:
I'll wait on changing anything because I'm still annoyed at all the merge conflicts